test/fork.js

```diff
@@ -38,11 +38,9 @@ test('rejects on error and streams output', function (t) {
 
 	fork(fixture('broken.js'))
 		.run()
-		.on('uncaughtException', function (data) {
-			t.true(/no such file or directory/.test(data.exception.message));
-		})
-		.catch(function () {
-			t.pass();
+		.catch(function (err) {
+			t.true(/exited with a non-zero exit code: 1/.test(err.message));
+			t.ok(err);
```

Comment:
That is because we have to let Node print it to stderr natively. We could clean up the output of our own tests by allowing `fork` to take `stdio` options like `child_process.fork` does (and pipe them to something else besides stderr for that particular test). That's a future PR, IMO.